config/crd/compute.gcp.crossplane.io_gkeclusters.yaml

```diff
@@ -38,342 +38,12 @@ spec:
-              - namespace
-              type: object
-            zone:
-              description: Zone specifies the name of the Google Compute Engine zone
-                in which this cluster resides.
-              type: string
-          required:
-          - numNodes
-          - providerRef
-          type: object
-        status:
-          description: A GKEClusterStatus represents the observed state of a GKECluster.
-          properties:
-            bindingPhase:
-              description: Phase represents the binding phase of a managed resource
-                or claim. Unbindable resources cannot be bound, typically because
-                they are currently unavailable, or still being created. Unbound resource
-                are available for binding, and Bound resources have successfully bound
-                to another resource.
-              enum:
-              - Unbindable
-              - Unbound
-              - Bound
-              - Released
-              type: string
-            clusterName:
-              description: ClusterName is the name of this GKE cluster. The name is
-                automatically generated by Crossplane.
-              type: string
-            conditions:
-              description: Conditions of the resource.
-              items:
-                description: A Condition that may apply to a managed resource.
-                properties:
-                  lastTransitionTime:
-                    description: LastTransitionTime is the last time this condition
-                      transitioned from one status to another.
-                    format: date-time
-                    type: string
-                  message:
-                    description: A Message containing details about this condition's
-                      last transition from one status to another, if any.
-                    type: string
-                  reason:
-                    description: A Reason for this condition's last transition from
-                      one status to another.
-                    type: string
-                  status:
-                    description: Status of this condition; is it currently True, False,
-                      or Unknown?
-                    type: string
-                  type:
-                    description: Type of this condition. At most one of each condition
-                      type may apply to a resource at any point in time.
-                    type: string
-                required:
-                - lastTransitionTime
-                - reason
-                - status
-                - type
-                type: object
-              type: array
-            endpoint:
-              description: Endpoint of the GKE cluster used in connection strings.
-              type: string
-            state:
-              description: State of this GKE cluster.
-              type: string
-          type: object
-      type: object
```

Comment:
Not exactly sure how to get generation to reflect the schemas of both `v1alpha3` and `v1beta1` here :thinking: